sdk/all/src/jmh/resources/otel.yaml

```diff
@@ -0,0 +1,27 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:5678
+
+processors:
+  batch:
+  queued_retry:
+
+extensions:
+  health_check:
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp:
+    endpoint: backend:8080
+    insecure: true
+
+service:
+  extensions: [health_check]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [batch, queued_retry]
+      exporters: [logging, otlp]
```

Comment:
we don't need otlp in here, do we? What's "backend:8080" pointing at, anyway?

Reply:
You're right. We don't need the otlp exporter here. It was a copy paste error, sorry. I'll remove it.